packages/presentation/lib/screens/movie_details/movie_details_screen.dart

```diff
@@ -28,9 +38,424 @@ class MovieDetailsScreen extends StatefulWidget {
+                                  Expanded(
+                                    child: Padding(
+                                      padding: const EdgeInsets.fromLTRB(
+                                        AppSizes.size4,
+                                        AppSizes.size4,
+                                        AppSizes.size0,
+                                        AppSizes.size4,
+                                      ),
+                                      child: Container(
+                                        alignment: Alignment.center,
+                                        decoration: BoxDecoration(
+                                          color: HomeScreenColors
+                                              .selectionActiveColor,
+                                          borderRadius: BorderRadius.circular(
+                                            HomeScreenSizes
+                                                .selectionBorderRadiusSize,
+                                          ),
+                                        ),
+                                        child: const Text(
+                                          'Detail',
+                                          style: SelectionButtonStyles
+                                              .activeButtonTextStyle,
+                                        ),
+                                      ),
+                                    ),
+                                  ),
+                                  Expanded(
+                                    child: Padding(
+                                      padding: const EdgeInsets.all(
+                                        AppSizes.size4,
+                                      ),
+                                      child: Container(
+                                        alignment: Alignment.center,
+                                        decoration: BoxDecoration(
+                                          color: Colors.transparent,
+                                          borderRadius: BorderRadius.circular(
+                                            HomeScreenSizes
+                                                .selectionBorderRadiusSize,
+                                          ),
+                                        ),
+                                        child: const Text(
+                                          'Reviews',
+                                          style: SelectionButtonStyles
+                                              .inactiveButtonTextStyle,
+                                        ),
+                                      ),
+                                    ),
+                                  ),
+                                  Expanded(
+                                    child: Padding(
+                                      padding: const EdgeInsets.fromLTRB(
+                                        0,
```

Comment:
if you have custom size for 0 use ur AppSizes.size0,

Reply:
Fixed.